.cirrus.yml

```diff
@@ -207,15 +212,19 @@ autoscan_task:
   <<: *ONLY_SONARSOURCE_QA
   eks_container:
     <<: *CONTAINER_DEFINITION
-    image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-latest
+    dockerfile: .cirrus/Dockerfile.jdk17And21
```

Comment:
Could you, please, put a comment with TODO, so we can use jdk 21 completely once SQ is compatible